webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+
+const clientPath = path.resolve(__dirname, 'client');
+const publicPath = path.resolve(__dirname, 'server/public');
+
+module.exports = {
+  mode: 'development',
+  resolve: {
+    extensions: ['.js', '.jsx']
+  },
```

Comment:
there are a lot of enhancements that **can** be done here for optimizing load times. if that interests you do it. if it doesnt watch some youtube about it. its a good thing to know about